src/main/scala/io/iohk/ethereum/jsonrpc/Web3Service.scala

```diff
@@ -25,4 +35,24 @@ class Web3Service {
     Future.successful(ClientVersionResponse(Config.clientVersion))
   }
 
+  def submitHashRate(req: SubmitHashRateRequest): Future[SubmitHashRateResponse] = {
```

Comment:
also I was thinking about creating something like `EthService.scala` for this methods